### ISSUE TITLE
running start on already debugged process gives exception

### ISSUE THREAD

**Issue by k4lizen** [pwndbg/pwndbg#2584]
### Description
If i run `start` when a process is already being debugged I get an exception. Only happens combined with pwntools? `run` doesn't give an exception. 

### Steps to reproduce
`exploit.py`:
```python
#!/usr/bin/env python
from pwn import *

exe = context.binary = ELF('/bin/sh', checksec=False)
context.terminal = "kitty @ launch --location=before --cwd=current --bias=65".split()

p = gdb.debug([exe.path])
pause()
```
```bash
~> python exploit.py
```
```
<the process is now being debugged at _start>
pwndbg> start
Temporary breakpoint 1 at 0x5c8710555980
╭─────────────────────────────── Traceback (most recent call last) ────────────────────────────────╮
│ -------------------------------/pwndbg/commands/__init__.py:188 in __call__                      │
│                                                                                                  │
│   185 │                                                                                          │
│   186 │   def __call__(self, *args: Any, **kwargs: Any) -> str | None:                           │
│   187 │   │   try:                                                                               │
│ ❱ 188 │   │   │   return self.function(*args, **kwargs)                                          │
│   189 │   │   except TypeError:                                                                  │
│   190 │   │   │   print(f"{self.function.__name__.strip()!r}: {self.function.__doc__.strip()}"   │
│   191 │   │   │   pwndbg.exception.handle(self.function.__name__)                                │
│                                                                                                  │
│ -------------------------------/pwndbg/commands/__init__.py:358 in _OnlyWithDbg                  │
│                                                                                                  │
│   355 │   │   def _OnlyWithDbg(*a: P.args, **kw: P.kwargs) -> Optional[T]:                       │
│   356 │   │   │   if pwndbg.dbg.is_gdblib_available():                                           │
│   357 │   │   │   │   if "gdb" in dbg_names:                                                     │
│ ❱ 358 │   │   │   │   │   return function(*a, **kw)                                              │
│   359 │   │   │   else:                                                                          │
│   360 │   │   │   │   if "lldb" in dbg_names:                                                    │
│   361 │   │   │   │   │   return function(*a, **kw)                                              │
│                                                                                                  │
│ -------------------------------/pwndbg/commands/start.py:83 in start                             │
│                                                                                                  │
│    80 │   │   │   continue                                                                       │
│    81 │   │                                                                                      │
│    82 │   │   gdb.Breakpoint(symbol, temporary=True)                                             │
│ ❱  83 │   │   gdb.execute(run, from_tty=False, to_string=True)                                   │
│    84 │   │   return                                                                             │
│    85 │                                                                                          │
│    86 │   # Try a breakpoint at the binary entry                                                 │
╰──────────────────────────────────────────────────────────────────────────────────────────────────╯
error: The "remote" target does not support "run".  Try "help target" or "continue".
If that is an issue, you can report it on https://github.com/pwndbg/pwndbg/issues
(Please don't forget to search if it hasn't been reported before)
To generate the report and open a browser, you may run `bugreport --run-browser`
PS: Pull requests are welcome
pwndbg> run
The "remote" target does not support "run".  Try "help target" or "continue".
pwndbg>
```
It's weird that `run` works. Is it also really a "remote" target?

### My setup
Gdb:      15.2
Python:   3.12.7
Pwndbg:   2024.08.29 build: ce954f74
Capstone: 5.0.1280
Unicorn:  2.1.0

**Comment by disconnect3d** [comment]
The "Only happens combined with pwntools" comes from the fact that pwntools use gdbserver under the hood.

**Comment by disconnect3d** [comment]
Fixed in #2600 :)

**Comment by k4lizen** [comment]
Ohhh I see, I didn't realize it was just a gdb error being raised. Thanks :>
